### DISCUSSION TITLE
How to refer to all the styleVariants inside of a GlobalStyle, etc?

### DISCUSSION THREAD

**Discussion by dawidseipold** [vanilla-extract-css/vanilla-extract#741] [Q&A]
Hi everyone! I wanted to ask about referencing all the ``styleVariants`` classes inside of the functions like ``GlobalStyle``. What I want to achieve is to have this code:
```globalStyle(`${variants} > a`, {
  color: 'purple',
});```
 refactored in such a way, that would allow me to pull all the classes at once, and not separately in a manual way.

**Comment by bwyx** [comment]
maybe this could work for you:

```ts
// create custom className
const variantSelector = 'VE'

// or use VE generated className
const variantSelector = style({})

const styleVariants = {
  small: [variantSelector, style({})],
  medium: [variantSelector, style({})],
  large: [variantSelector, ...],
}
```

that would allow you to reference all `styleVariants` at once.
```ts
// and then
globalStyle(`${variantSelector} > a`, { color: 'purple', ... })
```
